src/components/BottomBar/SkyBrowser/SkyBrowserSettings.jsx

```diff
@@ -264,26 +216,19 @@ class SkyBrowserSettings extends Component {
       </ToggleContent>);
```

Comment:
Move `);` to next line and fix whatever indentation problem that leads to this row not being indented 2 steps less